vignettes/hipercow.Rmd

```diff
@@ -26,10 +26,19 @@ Much of the difficulty of getting things running involves working out what your
 
 The `hipercow` package aims to remove some of this pain, with the aim that running a task on the cluster should be (almost) as straightforward as running things locally, at least once some basic setup is done.
 
-At the moment, this document assumes that we will be using the "Windows" cluster, which implies the existence of some future non-Windows cluster. Stay tuned.
-
 This manual is structured in escalating complexity, following the chain of things that a hypothetical user might encounter as they move from their first steps on the cluster through to running enormous batches of tasks.
 
+# Clusters and Platforms
+
+In this documentation, by *cluster*, we mean really a head-node. We support one of these at the moment: the DIDE cluster. In the future, we might also support the PBS cluster that ICT run, or we might create a pure linux cluster with a SLURM headnode for instance. But not yet.
+
+By *platform*, we mean operating system. You can use `hipercow` on your own computer, whether it runs Windows, Linux or MacOS. There are differences between these operating systems, mostly about how to refer to your DIDE network paths as
+they appear on your computer (ie, drive letters or mountpoints), and Hipercow tries to handle those for you without any fuss.
+
+Until recently, your jobs would then be run on Windows cluster nodes. However, unusually (and perhaps interestingly), our DIDE cluster runs Microsoft HPC, which can have both linux-based and windows-based nodes attached to it, using the same basic launch tools for either. This is interesting, since it can allow linux jobs to be run, but with a launch mechanism extremely similar to what you might already be familiar with from Hipercow.
```

Comment:
```suggestion
Until recently, your jobs would then be run on Windows cluster nodes. However, unusually (and perhaps interestingly), our DIDE cluster runs Microsoft HPC, which can have both linux-based and windows-based nodes attached to it, using the same basic launch tools for either. This is interesting, since it can allow linux jobs to be run, but with a launch mechanism extremely similar to what you might already be familiar with from `hipercow`.
```